WTWSMS/history/titles/d_mallorca.txt

```diff
@@ -3,4 +3,8 @@
 }
 477.2.9={
 	holder = 92
+}
```

Comment:
Here I think it could either be without holder or held by the next Vandal rulers.

Reply:
Done.